tests/failing/error.spec.lua

```diff
@@ -0,0 +1,7 @@
+return function()
+	describe("should fail", function()
+		it("failing", function()
+			error("Failed")
```

Comment:
we may want to add a test for `fail()` as well

Reply:
Done (plus a few more that I wrote but didn't add).